### DISCUSSION TITLE
Support for virtual boy

### DISCUSSION THREAD

**Discussion by itslemcy** [darthcloud/BlueRetro#135] [Ideas]
hi, are there any plans to make this project work with the virtual boy by any chance? 

i have a consolized vb and would love to use a vr controller. cheers!

**Comment by darthcloud** [comment]
Yes it is planned. I guess I should consolize mine, I got no stand anyway.

**Reply by itslemcy** [reply]
so just to try out i used the snes bin and it partially works obviously, a + b buttons are missing but everything else works great!

![image](https://user-images.githubusercontent.com/7327888/150607460-437e181f-a122-48b6-a505-62e2ea8f7de2.jpeg)

**Comment by darthcloud** [comment]
Ok here you go, I was waiting for my virtual tap to arrive but here a beta with VB firmware:

[firmware (1).zip](https://github.com/darthcloud/BlueRetro/files/7917166/firmware.1.zip)

BlueRetro-vb_spiffs.bin
2nd dpad on 2nd stick or C buttons if you got NSO N64

**Reply by itslemcy** [reply]
going to try this out right now, thanks!

**Reply by itslemcy** [reply]
@darthcloud tested it for 3/4 hours and it's flawless, i'm just going to put my hands forward by asking you if it's possible to trigger the color palette change with a key combo. 
here is the ref: http://furrtek.free.fr/virtualtap/manual_v1_en.pdf

**Reply by itslemcy** [reply]
hi! did you manage to consolize your vb and to check out my latest suggestion @darthcloud ?

**Comment by darthcloud** [comment]
Official support here:
https://github.com/darthcloud/BlueRetro/releases/tag/v1.3

See consolize instruction:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Consolize-Build-Instructions#virtual-boy

Mode is IO27 toggle mode and IO16 swap palette, you can hook those directly to VirtualTap without level shifter as VirtualTap is 3.3v.

See my youtube video:
https://www.youtube.com/watch?v=VbQTspncy0Y

**Reply by itslemcy** [reply]
A M A Z I N G
